examples/static-react/src/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * This example uses Express to serve static files
+ */
+
+const express = require( 'express' );
+const path = require( 'path' );
+
+const app = express();
+
+const PORT = process.env.PORT || 3000;
+const BASEURL = process.env.BASEURL || 'http://localhost/';
+
+/**
+ * Handle healthcheck requests
+ * 
+ * This is a requirement for any application running
+ * in the VIP environment:
+ * - must handle url "/cache-healthcheck?" with trailing question mark
+ * - must respond with 200 status
+ * - must only send a short response eg "ok"
+ * - must respond immediately, without delay
+ * - must prioritize this above other routes
+ * - should not execute significant code before, during, or after
+ * - must terminate the session
```

Comment:
I like this as direct advice that may not be obvious to those who only fixate on the 200 response. However, there may be cases where customers might want to do additional checks as part of the health check like check on the availability of Redis or external APIs.

I also worry that "terminate the session" might be confusing. Do we mean "end the request" or "end the connection" here?

```suggestion
 * - must only send a short response eg "ok"
 * - must respond immediately, without delay
 * - must prioritize this above other routes
 * - should not execute significant code before, during, or after
 * - must terminate the session
```
```suggestion
 * - must only send a short response eg "ok"
 * - must respond immediately, without delay
 * - must prioritize this above other routes
 * - should not execute significant code before, during, or after
 * - must terminate the session
```

Reply:
Fair enough. I think "end the response" is probably more apt and matches the terms used by Express in code in some contexts (`res.end()`).

Reply:
I tweaked this a bit more